src/components/VitalStat.vue

```diff
@@ -5,22 +5,22 @@
 <template>
   <div
     class="box">
-    <div class="name">{{stat.id}}:</div>
+    <div class="name">{{ stat }}:</div>
     <div
       class="points">
       <div 
         class="five"
         v-for="fives in numberOfFives"
         :key="fives">
         <span
-          v-for="i in (fives==numberOfFives && scale%5!=0) ? scale%5 : 5"
-          :key="i"
+          v-for="i in (fives==numberOfFives && scale%5!=0) ? scale%5 : 5"          
```

Comment:
?? to je akoze whitespace navyse?

Reply:
neviem co tam je nac, toto com nemenil, asi omylom...

Reply:
no pozri si to vo file changed... vypadalo to vcera ked som sa pozeral, ze tam su len nejake medzery na navyse